### PR TITLE
Change student auth

### PR DESCRIPTION
**Pull request by HiruNya**

### CODE REVIEW

**Comment by HiruNya** [comment]
Fixes #116

File: parsons-tool-backend/src/routes/api/student.js [hunk 1]
```diff
@@ -49,19 +43,27 @@ router.get('/problems/:group', async (req, res) => {
...
     console.log('[student.js]>', error);
-    res.status(500).json('An issue has occured on the server end');
+    res.status(500).json('An issue has occurred on the server end');
   }
 });
 
 // Get a particular student based on their email
 router.get('/:id', async (req, res) => {
   const { id } = req.params;
+  // const thisUser = await Users.find({ email: req.currentUser.email });
```
Comment (f-fromont) [inline parsons-tool-backend/src/routes/api/student.js:63]: should thes be commented out?

**Review by f-fromont — CHANGES_REQUESTED** [review]
Just a question about the commented out code, otherwise it looks good

**Review by f-fromont — APPROVED** [review]

**Comment by f-fromont** [comment]
issue made for commented code #129

**Pull request merged**